docs/getting-started.md

```diff
@@ -202,19 +202,47 @@ $ENV:AWS_SECRET_ACCESS_KEY=$awsCredentials.SecretAccessKey
 
 ## Deploying a cluster
 
+### Starting Minikube
+For security reasons, it's best to start a Minikube instance before using
+clusterctl to store your AWS credentials.
+
+``` shell
+minikube start
+```
+
+Then write the AWS credentials into the Minikube instance for use by Cluster API:
+
+``` shell
+minikube ssh 'mkdir -p .aws'
+clusterawsadm alpha bootstrap generate-aws-default-profile | minikube ssh 'cat > .aws/credentials'
+[default]
+aws_access_key_id = AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key = wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+region = eu-west-1
+
+
+^C
+```
+
+**NOTE**:
+> This will print the AWS credentials to the screen and then hang
+> (due to a restriction in Minikube).
+> This is OK, and you can cancel with Ctrl+C, but make sure to do this privately.
+
 ### Generating cluster manifests
 
-The shell script `generate-yaml.sh` in [clusterctl/examples](clusterctl/examples) can be used to generate the cluster manifests.
+The shell script `config/generate-yaml.sh` can be used to generate the
+cluster manifests, and copy your AWS environment credentials into the
+minikube environment.
 
-> The following command is valid in both Bash and PowerShell.
 
 ```bash
-sh -c "cd ./clusterctl/examples/aws && ./generate-yaml.sh"
+./config/generate-yaml.sh
 ```
 
-**NOTE**:
-> The generated manifests contain a copy of the AWS credentials.
-> Secure credentials storage is slated for a future release.
+Then edit `config/samples/out/cluster.yaml` and
+`config/samples/out/machine.yaml` for your SSH key and AWS region
+and any other customisations you want to make.
```

Comment:
Would it make sense if instead of using `config/samples/out` if we just moved the `out` directory to the root of the repo?

Reply:
Have no strong opinions on that, so happy to do this.